packages/fxa-settings/src/pages/SigninTokenCode/en.ftl

```diff
@@ -0,0 +1,14 @@
+## SigninTokenCode
+
+# String within the <span> element appears as a subheading
+signin-token-code-heading = Enter confirmation code<span> for your { -product-firefox-account }</span>
```

Comment:
I know this string is replicating a legacy string, but the way it's presented now some languages may translate this in a different order (with the span at the beginning). I would suggest a comment with an alternative similar to the ones provided for strings in ResetPassword. Perhaps:

```suggestion
# The subheading will appear below "Enter confirmation code", so if more appropriate in a locale, the string within the <span>, "for your { -product-firefox-account }" can stand alone as "{ -product-firefox-account }"
signin-token-code-heading = Enter confirmation code<span> for your { -product-firefox-account }</span>
```

Reply:
Good point. The key part of the span is that it appears on a separate line, so the following comment might provide more flexibility: 
```
# String within the <span> element appears on a separate line
# If more appropriate in a locale, the string within the <span>, "for your { -product-firefox-account }"
# can stand alone as "{ -product-firefox-account }"
```
This way, the span could be above or below "Enter confirmation code", either would be acceptable.

Reply:
Ah great, that's perfect!